src/chat/services/open-router.service.ts

```
@@ -41,16 +41,16 @@ export class OpenRouterService {
     private readonly httpService: HttpService,
     private readonly configService: ConfigService,
   ) {
-    this.apiKey = this.configService.getOrThrow<string>('OPEN_ROUTER_API_KEY');
+    this.apiKey = this.configService.getOrThrow<string>('LETSUR_AI_GATEWAY_API_KEY');
```

ESLint

[error] Replace `'LETSUR_AI_GATEWAY_API_KEY'` with `⏎······'LETSUR_AI_GATEWAY_API_KEY',⏎····`

(prettier/prettier)